replicatedhq/ship
# Fix diff doesn't work properly after refresh

What I Did
------------
- Fix #375 

How I Did it
------------
- Previously, diffs were only generated when an overlay was saved
- Diffs are now generated if an existing `patch` is found and if the `modified` is empty

- Testing related:
  - Add mocks for `AceEditorHOC` and `DiffEditor`
  - Change the way the `ref` is defined for `aceEditorOverlay` - [for reference](https://github.com/airbnb/enzyme/issues/1407)

How to verify it
------------
- Navigate through UI
- Test

Description for the Changelog
------------
- Fix #375 


Picture of a Boat (not required but encouraged)
------------
⛵️
